build.gradle

```diff
@@ -1,37 +1,63 @@
-//------------------------------------------- PLUGIN DEFINITION -----------------------------------------------//
+/**
+Template file for classes using GradleRIO
+@author Jaci
+*/
 
-plugins {
-    id "java-gradle-plugin"
-    id "eclipse"
-    id "jacoco"
+//----------------------------------------- BUILD SCRIPT ------------------------------------------------//
+
+buildscript {
+  repositories {
+    mavenLocal()
+	mavenCentral()
+	maven {
+		name = "GradleRIO"
+		url = "http://dev.imjac.in/maven"
+	}
+  }
+  dependencies {
+    classpath group: 'jaci.openrio.gradle', name: 'GradleRIO', version: '+'			//Change this line if you wish to Update GradleRIO
+//    compile 'org.apache.commons:commons-collections4:4.1'
+//    testCompile group: 'junit', name: 'junit', version: '4.+'
+  }
 }
```

Comment:
Same as below. How do the other dependencies get pulled in? like say commons-collections? I dont wanna have this commented piece here because i want to make sure we know how to import external dependencies. This is a good time to find out.